client/src/components/Panel/index.ts

```diff
@@ -0,0 +1,119 @@
+export class Panel {
+  type: string; // Type of the panel; will generally match the Python-side panel class name (e.g., 'FieldPanel', 'PanelGroup')
+  prefix: string; // Prefix for the panel's HTML elements (e.g., 'field-')
```

Comment:
Attributes can also have docstrings

```suggestion
  /** Type of the panel; will generally match the Python-side panel class name (e.g., `FieldPanel`, `PanelGroup`) */
  type: string;
  /** Prefix for the panel's HTML elements (e.g., `field-`) */
  prefix: string;
```